src/components/atoms/AccessDeniedModal/AccessDeniedModal.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useCallback, useState } from "react";
+import { Button, Modal } from "react-bootstrap";
+import { useHistory } from "react-router-dom";
+import { venueLandingUrl } from "utils/url";
+
+interface AccessDeniedModalProps {
+  show?: boolean;
+  venueId: string;
+  venueName: string;
+}
+
+export const AccessDeniedModal: React.FC<AccessDeniedModalProps> = ({
+  show = true,
+  venueId,
+  venueName,
+}) => {
+  const [isVisible, setIsVisible] = useState(show);
+  const history = useHistory();
+
+  const hide = useCallback(() => {
+    setIsVisible(false);
+  }, []);
+
+  const confirm = useCallback(() => {
+    history.push(venueLandingUrl(venueId));
+    hide();
+  }, [history, venueId, hide]);
```

Comment:
Instead of using a `<Button>` and passing this handler to it, we could potentially just use react-router's `<Link>` (styled as a button if that's what we want it to look like):

- https://reactrouter.com/web/api/Link

Reply:
https://reactrouter.com/web/api/Link/

Link doesn't seem to accept a callback function and I can't hide the modal using it.

Reply:
That’s fair. I think I may have missed that requirement in my first pass.